### PR TITLE
Apply edits from IRSG reviews.

### PR DESCRIPTION
**Pull request by chris-wood**
This is based on the reviews here: https://datatracker.ietf.org/doc/draft-irtf-cfrg-randomness-improvements/ballot/irsg/

### CODE REVIEW

File: draft-irtf-cfrg-randomness-improvements.md [hunk 1]
```diff
@@ -116,37 +118,39 @@ normative:
...
+TLS in particular uses CSPRNGs to generate several values: session IDs,
 ephemeral key shares, and ClientHello and ServerHello random values. CSPRNG failures
 such as the Debian bug described in {{DebianBug}} can lead to insecure TLS connections.
 CSPRNGs may also be intentionally weakened to cause harm {{DualEC}}.
 Initial entropy sources can also be weak or broken, and that would lead to insecurity
-of all CSPRNG instances seeded with them.
-In such cases where CSPRNGs are poorly implemented or insecure, an adversary may be
-able to predict its output and recover secret key material used to protect the connection.
+of all CSPRNG instances seeded with them. In such cases where CSPRNGs are poorly
+implemented or insecure, an adversary Adv may be distinguish its output from a random
```
Comment (smyshsv) [inline draft-irtf-cfrg-randomness-improvements.md:142]: "... may be able to distinguish ..." or "may distinguish"

File: draft-irtf-cfrg-randomness-improvements.md [hunk 2]
```diff
@@ -155,13 +159,13 @@ is available and CSPRNG randomness guarantees are dubious, or to provide stronge
 about possible future issues with the randomness. Roughly, the security properties provided
 by the proposed construction are as follows:
 
-1. If the CSPRNG works fine, that is, in a certain adversary model the CSPRNG output is
+1. If the CSPRNG works fine, that is, in a certain adversarial model the CSPRNG output is
```
Comment (smyshsv) [inline draft-irtf-cfrg-randomness-improvements.md:162]: I wouldn't change "adversary model" to "adversarial model"; "adversary model" seems to be a more common term

File: draft-irtf-cfrg-randomness-improvements.md [hunk 3]
```diff
@@ -155,13 +159,13 @@ is available and CSPRNG randomness guarantees are dubious, or to provide stronge
...
 indistinguishable from a truly random sequence, then the output of the proposed construction
-is also indistinguishable from a truly random sequence in that adversary model.
-2. An adversary Adv with full control of a (potentially broken) CSPRNG and able to
-observe all outputs of the proposed construction, does not obtain any non-negligible
-advantage in leaking the private key, modulo side channel attacks.
-3. If the CSPRNG is broken or controlled by adversary Adv, the output of the proposed construction
+is also indistinguishable from a truly random sequence in that adversarial model.
```
Comment (smyshsv) [inline draft-irtf-cfrg-randomness-improvements.md:164]: same here

File: draft-irtf-cfrg-randomness-improvements.md (hunk 1 above)
Comment (chris-wood) [reply draft-irtf-cfrg-randomness-improvements.md]: Whoops -- fixed!

**Comment by chris-wood** [comment]
@smyshsv fixed based on your suggestions -- please have a look!

**Comment by cascremers** [comment]
Looks good! 

Minor suggestion: replace "(for salt as a key)" with "(for salt as a key of length M)" to make the role of M clearer in this requirement.

**Pull request merged**